docs/service-mesh/01-01-service-mesh.md

```diff
@@ -5,4 +5,8 @@ title: Overview
 Kyma Service Mesh is the component responsible for service-to-service communication, proxying, service discovery, traceability, and security. Kyma Service Mesh
 is based on [Istio](https://istio.io/docs/concepts/what-is-istio/) open platform. The main principle of Kyma Service Mesh operation is the process of injecting Pods of every service with an Envoy - a sidecar proxy which intercepts the communication between the services and regulates it by applying and enforcing the rules you create. Kyma [Dex](https://github.com/coreos/dex), which is also a part of the Service Mesh, allows you to integrate any [OpenID Connect](https://openid.net/connect/)-compliant identity provider or a SAML2-based enterprise authentication server with your solution.
 
-See this [Istio diagram](https://istio.io/docs/concepts/what-is-istio/arch.svg) to understand the relationship between the Istio components and Services.
+By default, the Kyma implementation of Istio has [mutual TLS (mTLS)](https://istio.io/docs/tasks/security/mutual-tls/) enabled and injects an Istio sidecar Pod to every service. If you disable sidecar injection in a service, you must manage its traffic by creating appropriate [Destination Rules](https://istio.io/docs/reference/config/networking/v1alpha3/destination-rule/) and [Authentication Policies](https://istio.io/docs/reference/config/istio.authentication.v1alpha1/). 
+
+>**NOTE:** The only part of the Service Mesh that doesn't have mTLS enabled is the Mesh Control Plane.
```

Comment:
I would just say that service mesh control plane does not have mTLS because we have disabled mTLS in a few kyma components as well so saying "only" is not true